### DISCUSSION TITLE
Event Reminders

### DISCUSSION THREAD

**Discussion by AlejandroClaudio** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#4] [Ideas]
We could add a system that can send reminders and updates about upcoming events and resource bookings via push notifications or email. It also could notify users of any changes to event details, such as time, location, or cancellation. This can help people stay up to date with all of the constant change of information.

**Comment by Kiarancole** [comment]
Definitely a necessary feature!  Expanding on this, a personalized notification system would be a good idea, as a way for users to choose which notifications to receive based on their interests or specific event categories. This would ensure users aren't overwhelmed with irrelevant updates as tends to happen with Cartero emails.

**Comment by AngelUPR** [comment]
I think the notifications should be through the app and avoid email as much as possible because a lot of email tend to be lost due to cartero's overwhelmance of emails

**Comment by estefania-2817** [comment]
I believe this can be a great feature, especially if there is some type of "like" or "save" button that suggests that you are interested in a particular event. Therefore, keep you informed of any changes of the event's details.

**Comment by Ojani** [comment]
Web push is a technology which allows to send push notifications through web apps. It is even supported on iPhone since iOS 16.4 as well as pc and android. I think we could use this.

**Reply by LuisJCruz** [reply]
This definitely sounds like the way to go.
